apps/prairielearn/elements/pl-string-input/pl-string-input.py

```diff
@@ -107,7 +107,7 @@ def render(element_html: str, data: pl.QuestionData) -> str:
                     "Leading and trailing spaces will be left as part of your answer."
                 )
             case _:
-                raise Exception("Should never reach here.")
+                raise NotImplementedError("Should never reach here.")
```

Comment:
Seems like another good case for `assert_never(...)`?

Reply:
Resolved in [b62be5d](https://github.com/PrairieLearn/PrairieLearn/pull/11104/commits/b62be5d6c6eb998f5a4a887abe0fddc2728f445e).